src/main/java/fr/quatrevieux/araknemu/game/player/characteristic/PlayerLife.java

```diff
@@ -32,11 +32,14 @@
     final private GamePlayer player;
     final private Player entity;
 
+    private long lifeRegeneration;
```

Comment:
- lifeRegenerationStart
- Add a docblock to indicate that it's in microseconds and the value should be set to 0 to disable regen
- Regroup with lifeRegenerationSpeed